mypy/checkmember.py

```diff
@@ -898,7 +897,7 @@ def f(self: S) -> T: ...
             selfarg = get_proper_type(item.arg_types[0])
             if subtypes.is_subtype(dispatched_arg_type, erase_typevars(erase_to_bound(selfarg))):
                 new_items.append(item)
-            elif isinstance(selfarg, ParamSpecType):
+            elif isinstance(selfarg, (ParamSpecType, TupleType)):
```

Comment:
Bit of a lazy hack to keep `testTypeTupleCall` and `testTypeNamedTupleCall` green

Reply:
This looks suspicious. `ParamSpecType` doesn't really make sense here (and it is unlikely it can get there without previously emitting an error), but `TupleType` is relatively common self-type for various user defined tuple types. Is it needed because type erasure touches some unhandled/incorectly handled corner case in `is_subtype()`? If yes, then that should be fixed instead.

Reply:
Yeah, needs a change in is_subtype: https://github.com/python/mypy/pull/16073